docs/de/README.md

```diff
@@ -61,8 +61,9 @@ Bedeutung der Optionen im Konfigfile:
 
 - Erklärung zu den States unter ical.0.events.0 
   - Das Event im Pfad ical.0.events.0.later wird auf true gesetzt, wenn es heute noch stattfindet aber noch nicht begonnen ist
-  - Das Event im Pfad ical.0.events.0.now wir auf true gesetzt, wenn es aktuell aktiv ist
+  - Das Event im Pfad ical.0.events.0.now wird auf true gesetzt, wenn es aktuell aktiv ist
   - Das Event im Pfad ical.0.events.0.today wird auf true gesetzt, wenn das Event am heutigen Tag aktiv ist
+  - Hinweis: Termine der Vortage werden aktuell nicht angezeigt, dies ist aber auf der ToDo-Liste

```

Comment:
In der Doku sollten nur Dinge beschrieben sein, welche auch implementiert sind. Auch wenn ein Featurewunsch existiert kann man daraus nicht ableiten, wann und ob er implementiert wird.